bundles/org.openhab.binding.velbus/src/main/java/org/openhab/binding/velbus/internal/handler/VelbusThingHandler.java

```diff
@@ -152,7 +152,14 @@ private void initializeThing(ThingStatus bridgeStatus) {
                 if (subAddress != null) {
                     subAddresses[i] = hexToByte(subAddress);
                 } else {
-                    subAddresses[i] = (byte) 0xFF;
+                    if (getConfig().get(propertyKey) != null) {
+                        subAddress = getConfig().get(propertyKey).toString();
+                        if (subAddress != null) {
```

Comment:
This can’t be `null`, can it? Since the property is `required = "true"`…? And either way, wouldn’t it be better to somehow report that this thing is misconfigured instead of moving on with a guess of what the users’ intent was?